modules/core/src/main/java/org/apache/ignite/internal/processors/cache/ClusterCachesInfo.java

```diff
@@ -1040,6 +1043,8 @@ else if (encMgr.masterKeyDigest() != null &&
 
         DynamicCacheDescriptor old = registeredCaches.put(ccfg.getName(), startDesc);
 
+        cacheIdToNameMapping.putIfAbsent(startDesc.cacheId(), ccfg.getName());
```

Comment:
Why it will be absent? Can we do an assert that it is successful? Or maybe a simple put?

Reply:
Fixed, thanks